packages/asset-service/src/service/TrustWalletService.ts

```diff
@@ -17,7 +17,10 @@ export const generateTrustWalletUrl = (assetId: AssetId) => {
     url += `/assets/`
     switch (chainNamespace) {
       case CHAIN_NAMESPACE.Evm:
-        url += Web3.utils.toChecksumAddress(assetReference)
+        // Skip checksum if we're hitting native ETH
+        if (assetReference !== '60') {
+          url += Web3.utils.toChecksumAddress(assetReference)
+        }
```

Comment:
An ugly bit, this logic doesn't work for native ETH. Don't wanna mess with rewriting the asset generation logic in this PR but probably worth doing as a follow-up

Reply:
This is actually totally fine, 60 SLIP-44 is all we need to detect a native EVM asset, we could do something fancier like check the `assetId` against a list of native asset IDs but this is absolutely fine and future-proof